src/pages/community-champions.js

```diff
@@ -42,6 +49,76 @@ const StyledHeroBanner = styled(HeroBanner)`
     }
 `;
 
+const ContentContainer = styled('div')`
+    margin: 0 ${size.xxlarge};
+    @media ${screenSize.upToLarge} {
+        margin: 0 ${size.default};
+    }
+`;
+
+const Title = styled(H1)`
+    margin: ${size.xlarge} 0;
+    word-wrap: break-word;
```

Comment:
no. i initially added this because i plan to reuse this title component for other section titles and wanted to make sure we wouldn't have to scroll horizontally to see longer words but looks like this won't be an issue since none of the words in the section titles are too long. i'll get rid of this 🙂